builtin/providers/azurerm/resource_arm_virtual_machine_scale_set.go

```diff
@@ -74,6 +74,13 @@ func resourceArmVirtualMachineScaleSet() *schema.Resource {
 				Optional: true,
 			},
 
+			"single_placement_group": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+				ForceNew: true,
```

Comment:
Understood :) I guess if we need to, we can follow up with more documentation about this.

It's important a user knows that a scaleset will be recreated if this value changes